common/lib/capa/capa/responsetypes.py

```diff
@@ -1344,17 +1552,19 @@ def setup_response(self):
             return
         # end of backward compatibility
 
-        correct_answers = [self.xml.get('answer')] + [el.text for el in self.xml.findall('additional_answer')]
+        # XML compatibility note: in 2015, additional_answer switched to having a 'answer' attribute.
+        # See make_xml_compatible in capa_problem which translates the old format.
+        correct_answers = ([self.xml.get('answer')] + 
+            [element.get('answer') for element in self.xml.findall('additional_answer')])
```

Comment:
1557: W291 trailing whitespace
1558: E128 continuation line under-indented for visual indent

Would recommend doing,
```python
var = (
    first +
    second
)
```